gomock/matchers.go

```diff
@@ -176,6 +176,24 @@ func (am allMatcher) String() string {
 	return strings.Join(ss, "; ")
 }
 
+type lenMatcher struct {
+	i int
+}
+
+func (m lenMatcher) Matches(x interface{}) bool {
+	v := reflect.ValueOf(x)
+	switch v.Kind() {
+	case reflect.Array, reflect.Chan, reflect.Map, reflect.Slice, reflect.String:
```

Comment:
curious, does this cover `*[]string` and `[]string` ?

Reply:
This would not, do you think we should support that?

Reply:
I don't think it's necessarily needed. If it supports `[]string` but not `*[]string` the user can just dereference.